packages/video_player/elinux/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(PLUGIN_NAME "video_player_elinux_plugin")
 find_package(PkgConfig)
 pkg_check_modules(GLIB REQUIRED glib-2.0)
 pkg_check_modules(GSTREAMER REQUIRED gstreamer-1.0)
+pkg_check_modules(GSTREAMER_GL REQUIRED gstreamer-gl-1.0)
```

Comment:
If you want to use `USE_EGL_IMAGE_DMABUF`, we should use it in CMakeLists.txt to cut dependency to the library too.

Reply:
Thank you for your comment.
I agree with you.
But I didn't have any good ideas for doing that.
I tried the following code but it didn't work:
```
if(USE_EGL_IMAGE_DMABUF)
  pkg_check_modules(GSTREAMER_GL REQUIRED gstreamer-gl-1.0)
endif()
```
The following options are given when building an application:
`--target-compiler-flags="-DUSE_EGL_IMAGE_DMABUF"`

Is there any good way to do this?

Reply:
> The following options are given when building an application:
> --target-compiler-flags="-DUSE_EGL_IMAGE_DMABUF"

Sorry I didn't explain it enough, but I didn't intend to use that build option. I meant USE_EGL_IMAGE_DMABUF can be added in CMakeLists.txt in user's project dir if users want to use it. Not dynamic way, it's static one.

Reply:
Thank you for your advice.
Do you mean to do it this way?
If users want to use `USE_EGL_IMAGE_DMABUF`, add the following code to <user's project>/elinux/CMakeLists.txt.
```Shell
add_definitions(-DUSE_EGL_IMAGE_DMABUF)
set(USE_EGL_IMAGE_DMABUF "on")
```

Reply:
Exactly.